pipeline.py

```diff
@@ -51,19 +69,30 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        path_to_save = Path(self.path_to_raw_txt_data)
```

Comment:
if you want to explicitly make it a path - do it in the _init_ method

Reply:
I redid it into a list that sorted by files' indices